resources/data/dictionary/outputs.csv

```diff
@@ -0,0 +1,377 @@
+55,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_clothes_washer_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Clothes Washer,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+56,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_cooling_fans_pumps_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Cooling Fans/Pumps,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+57,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_cooling_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Cooling,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+58,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_dehumidifier_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Dehumidifier,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+59,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_dishwasher_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Dishwasher,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+60,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_electric_vehicle_charging_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Electric Vehicle Charging,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+61,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_freezer_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Freezer,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+62,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_generator_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Generator,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+63,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_heating_fans_pumps_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Heating Fans/Pumps,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+64,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_heating_heat_pump_backup_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Heating Heat Pump Backup,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+65,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_heating_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Heating,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
+66,report_simulation_output.emissions_co_2_e_lrmer_mid_case_15_electricity_hot_tub_heater_lb,lb,Emissions: CO2e: LRMER_MidCase_15: Electricity: Hot Tub Heater,lb,84,Scenario emissions for this Electricity end use only (one row per end use)
```

Comment:
I don't know what the right answer is, but it seems misleading to have "lrmer_mid_case" as part of the name here since it will be specific to which scenario(s) are in the yaml.

Also, all the "(one row per end use)" should be removed since you are already enumerating all of the end uses here.

Reply:
Yeah, I've thought about the scenario name issue, too. I put an attempted [clarifying note](https://github.com/NREL/resstock/blob/data-dictionary/docs/read_the_docs/source/outputs/optional_outputs.rst#emissions) in the docs about it.

Good catch on the "one row per end use" thing.